src/nvim/normal.c

```diff
@@ -3612,6 +3626,108 @@ void scroll_redraw(int up, long count)
 }
 
 /*
```

Comment:
Multiline `/**/` comments are not allowed, use `///`. How did linter accept this?